secrets/gs_master_passwd.txt

```diff
@@ -0,0 +1 @@
+gerlsSnFd6SmM
```

Comment:
I'd rename GS_MASTER into GEOSERVER_PRIVILEGED_USER to make it clear it's the same

Reply:
Sorry, I should have seen that when reviewing https://github.com/georchestra/georchestra/pull/3269